demos/supabase-todolist/pubspec.yaml

```diff
@@ -10,9 +10,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  powersync_attachments_helper: ^0.1.4
+  powersync_attachments_helper:
+    path: ../../packages/powersync_attachments_helper
 
-  powersync: ^1.2.0
+  powersync:
+    path: ../../packages/powersync
```

Comment:
I believe `melos` should do this linking automatically, or is there some issue for that?